### ISSUE TITLE
Check box to exclude local addresses from proxy

### ISSUE THREAD

**Issue by mattdale77** [mullvad/browser-extension#179]
I am constantly connecting and disconnecting the proxy as I have to access local services a lot. It would be great if there was a check box to exclude the local network or a way to exclude certain subnets or IPs. I've seen similar requests but they more involve passthrough from firefox and I believe you're not going to use the firefox proxy configuration.

**Comment by ruihildt** [comment]
In the current future implementation of the proxy, localhost are excluded from proxying.

Does it make sense to ever proxy localhost?

**Comment by mattdale77** [comment]
> In the current future implementation of the proxy, localhost are excluded from proxying.
> 
> Does it make sense to ever proxy localhost?

I don't think so. By the same rational I would suggest it never makes sense to proxy local networks. 192.168.X.X, 172.16.X.X, 10.X.X.X
